docs/src/components/ComponentStructure/index.tsx

```diff
@@ -32,8 +32,8 @@ const StyledTab = styled.button<{ active: boolean }>`
   ${({ theme, active }) => css`
     padding: 14px ${theme.orbit.spaceMedium};
     margin-bottom: -1px;
-    border-top-left-radius: ${theme.orbit.borderRadiusLarge};
-    border-top-right-radius: ${theme.orbit.borderRadiusLarge};
+    border-top-left-radius: ${theme.orbit.borderRadius150};
```

Comment:
We want to include the new tokens in `styled-components` as well, right?
@DSil

Reply:
We have to. Styled components use the tokens directly. If we remove the tokens, styled components can't get the values from it. That is why we need to update the tokens there as well.